airflow/providers/google/cloud/operators/dataplex.py

```diff
@@ -610,3 +613,848 @@ def execute(self, context: Context) -> None:
+
+
+class DataplexRunDataQualityScanOperator(GoogleCloudBaseOperator):
+    """
+    Runs an on-demand execution of a DataScan.
+
+    :param project_id: Required. The ID of the Google Cloud project that the lake belongs to.
+    :param region: Required. The ID of the Google Cloud region that the lake belongs to.
+    :param data_scan_id: Required. Data Quality scan identifier.
+    :param api_version: The version of the api that will be requested for example 'v1'.
+    :param retry: A retry object used  to retry requests. If `None` is specified, requests
+        will not be retried.
+    :param timeout: The amount of time, in seconds, to wait for the request to complete.
+        Note that if `retry` is specified, the timeout applies to each individual attempt.
+    :param metadata: Additional metadata that is provided to the method.
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :param asynchronous: Flag informing that the Dataplex job should be run asynchronously.
+        This is useful for submitting long-running jobs and
+        waiting on them asynchronously using the DataplexDataQualityJobStatusSensor
+    :param fail_on_dq_failure: If set to true and not all Data Quality scan rules have been passed,
+        an exception is thrown. If set to false and not all Data Quality scan rules have been passed,
+        execution will finish with success.
+    :param: result_timeout: Value in seconds for which operator will wait for the Data Quality scan result.
+            Throws exception if there is no result found after specified amount of seconds.
+
+    :return: Dataplex Data Quality scan job id.
+    """
+
+    template_fields = ("project_id", "data_scan_id", "impersonation_chain")
+
+    def __init__(
+        self,
+        project_id: str,
+        region: str,
+        data_scan_id: str,
+        api_version: str = "v1",
+        retry: Retry | _MethodDefault = DEFAULT,
+        timeout: float | None = None,
+        metadata: Sequence[tuple[str, str]] = (),
+        gcp_conn_id: str = "google_cloud_default",
+        impersonation_chain: str | Sequence[str] | None = None,
+        asynchronous: bool = False,
+        fail_on_dq_failure: bool = False,
+        result_timeout: float = 60 * 10,
```

Comment:
In sensor you have this variable as
result_timeout: float = 60.0 * 10,

Reply:
fixed, thanks!